setup.cfg

```diff
@@ -12,9 +12,6 @@ source-dir = docs/
 build-dir = docs/_build
 all_files = 1
 
-[bdist_wheel]
-universal = 1
-
```

Comment:
Just a few packages missing after this PR:
 
```
$ rg -i "universal = 1"  
reana-workflow-engine-snakemake/setup.cfg
16:universal = 1

reana-message-broker/setup.cfg
16:universal = 1

reana-workflow-engine-serial/setup.cfg
16:universal = 1

reana-workflow-engine-cwl/setup.cfg
16:universal = 1

pytest-reana/setup.cfg
18:universal = 1
```

Should we go ahead so we can close https://github.com/reanahub/reana/issues/559?